sdk/core/azure-core-tracing-opentelemetry/src/main/java/com/azure/core/tracing/opentelemetry/OpenTelemetryUtils.java

```diff
@@ -21,11 +21,11 @@
 class OpenTelemetryUtils {
     private static final ClientLogger LOGGER = new ClientLogger(OpenTelemetryUtils.class);
 
-    private static final Map<String, String> ATTRIBUTE_MAPPING_V1_17_0 = getMappingsV1200();
+    private static final Map<String, String> ATTRIBUTE_MAPPING_V1_17_0 = getMappingsV1700();
     static final String SERVICE_REQUEST_ID_ATTRIBUTE = "serviceRequestId";
     static final String CLIENT_REQUEST_ID_ATTRIBUTE = "requestId";
 
-    private static Map<String, String> getMappingsV1200() {
+    private static Map<String, String> getMappingsV1700() {
```

Comment:
Should we just make this name more generic, such as `getLatestSupportedMappings`?

Reply:
Also nit: at this time this mapping is so small I wonder if this would be better as a sorted `String[]` where `Arrays.binarySearch` could be used. Should make it so the entire search space is collocated whereas `Map` could result in cache misses.

Reply:
great point, I actually think that with 4 items, simple switch is enough :)